layouts/partials/home/hero.html

```diff
@@ -2,42 +2,27 @@
 {{- $acronym     := .Site.Params.home.acronym | markdownify }}
 {{- $description := .Site.Params.home.description | markdownify }}
 {{- $spiffeLogo  := "img/logos/spiffe/icon/color/spiffe-icon-color.svg" | relURL }}
+{{- $spiffeTurtleLogo  := "img/spiffe-turtle.svg" | relURL }}
 {{- $spireLogo   := "img/logos/spire/icon/color/spire-icon-color.svg" | relURL }}
 <section class="hero is-medium is-light">
-  <div class="hero-body has-text-centered">
-    <div class="container">
-      <img class="title is-hero-logo" src="/img/logos/spiffe/horizontal/color/spiffe-horizontal-color.png">
-
-      <br />
-
-      <p class="project-title is-size-4 is-size-4-mobile has-text-weight-bold">
-        Secure Production Identity Framework for Everyone
-      </p>
-      <p class="subtitle is-size-3 is-size-5-mobile has-text-weight-light">
-        {{ $description }}
-      </p>
+  <div class="hero-body is-flex is-align-items-center is-columns-centered p-3 flex-column-mobile-reverse">
+    <div class="">
+      <div class="text-container">
+        <h4 class="title is-2 is-size-4-mobile">Universal identity control plane for distributed systems</h4>
+        <p class="subtitle is-5 subtitle-text">Spiffe and SPIRE provide strongly attested, cryptographic identities to workloads across a wide variety of platforms</p>
```

Comment:
This should read `SPIFFE`, all uppercase.

Reply:
Alright. Would fix up now